examples/react-router/app/components/lists/show.jsx

```diff
@@ -0,0 +1,61 @@
+import Connect from '../../../../../src/addons/connect'
+import ItemForm from './parts/item-form'
+import Items, { addItem, removeItem } from '../../resources/items'
```

Comment:
Yeah, it's clear if you think about it... but you have to think. People looking at the example are coming in with no clue what they're doing, and would benefit from things being a little spelled out.